Content.Server/Administration/ControlMob.cs

```diff
@@ -0,0 +1,82 @@
+using Content.Server.GameObjects.Components.Mobs;
+using Content.Server.Mobs;
+using Content.Server.Players;
+using Robust.Server.Interfaces.Console;
+using Robust.Server.Interfaces.Player;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Interfaces.GameObjects;
+using Robust.Shared.IoC;
+using Robust.Shared.Localization;
+
+namespace Content.Server.Administration
+{
+    class ControlMob : IClientCommand
+    {
+        public string Command => "controlmob";
+        public string Description
+        {
+            get
+            {
+                var localizationManager = IoCManager.Resolve<ILocalizationManager>();
+                return localizationManager.GetString("Transfers user mind to the specified entity.");
```

Comment:
You can use the static `Loc` class to get access to localization instead of manually resolving with IoC.

Reply:
Done, didn't know about that class